Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -589,6 +591,25 @@ extension ProjectDescription.Settings {
         if !swiftFlags.isEmpty {
             settingsDictionary["OTHER_SWIFT_FLAGS"] = .array(["$(inherited)"] + swiftFlags)
         }
+        if let cLanguageStandard = packageInfo.cLanguageStandard {
+            settingsDictionary["GCC_C_LANGUAGE_STANDARD"] = .string(cLanguageStandard)
+        }
+        if let cxxLanguageStandard = packageInfo.cxxLanguageStandard {
+            settingsDictionary["CLANG_CXX_LANGUAGE_STANDARD"] = .string(cxxLanguageStandard)
+        }
+        if let swiftLanguageVersion = packageInfo.swiftLanguageVersions?
```

Comment:
This method has many concerns. I'd recommend extracting this logic into either a private method in this mapper, or a extension on an Array when Element is `Version` and making `swiftLanguageVersions` of type `[Version]` instead of `[String]`.

Reply:
extracted to fileprivate function of PackageInfo